### PR TITLE
Create gt-rpj.json fixes #374

### LINKED ISSUE #374: PROPOSAL: new list for Guatemala
## Proposal

**The name of the list (or the organization that manages the list)**
Registro de las Personas Juridicas, Ministerio de Gobernacion. 

**A suggested code for the list (optional)**
RPJ?

**A short description of the list**
A list of all organisations or legal entities in Guatemala

**The country or countries that the list covers**
Guatemala

**The legal form or organizations that the list covers**
All, it seems.

**Any specific sectors that the list covers**
All?

**A URL for information on the list, and a URL for looking up identifiers (if available)**
None that I'm aware of.

**Any information on available open data copies of this list**
None

**One or more examples of identifier from this list, and how you found them**
Plan International Guatemala: # 93 254 42 (as provided by the organisation). The official document is attached. It seems like this ID is a combination of several numbers mentioned in the document. Perhaps this list needs some double checking.
[certificacion de inscripcion de  Plan International.pdf](https://github.com/org-id/register/files/3724696/certificacion.de.inscripcion.de.Plan.International.pdf)

### PR DESCRIPTION
**Pull request by kd-ods**

### CODE REVIEW

File: lists/gt/gt-rpj.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "features": [
+      "date_of_registration",
+      "registration_number",
+      "activities_sector_code"
+    ],
+    "licenseStatus": "no_license",
+    "licenseDetails": ""
+  },
+  "meta": {
+    "source": "Government websites, legal documents provided by organisation",
```
Comment (siwhitehouse) [inline lists/gt/gt-rpj.json:44]: I'm unable to find a record for Plan International in any of the pdfs on the site. Nor can I find any Oxfam, Costa Foundation, CAFOD or ActionAid UK on the current NGO list, despite them being listed as working in Guatemala according to https://ngoexplorer.org/country/GTM

Am I missing something?

**Review by siwhitehouse — CHANGES_REQUESTED** [review]

File: lists/gt/gt-rpj.json (hunk 1 above)
Comment (kd-ods) [reply lists/gt/gt-rpj.json:44]: I can't see anything either. On the submission form to register an entity there are 7 categories, including a foreign entities category. On the data publication page there are PDFs for the other 6 categories, and they were published in 2018. Maybe the registration of foreign entities has only been required recently?

I don't think there's much more we can do here.

File: lists/gt/gt-rpj.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity"
+  ],
+  "sector": [],
+  "code": "GT-RPJ",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "secondary",
```
[the remark this answers is not in the document]
Comment (kd-ods) [reply lists/gt/gt-rpj.json:21]: Will update as you suggest.

**Review by siwhitehouse — APPROVED** [review]
Thanks for making those changes. I think this is a good entry for org-id.

**Pull request merged**